src/components/BackToTopButton/BackToTopButton.svelte

```diff
@@ -0,0 +1,158 @@
+<script>
+	import { onMount } from 'svelte';
+
+	// All sizing props can be passed with or without 'px' unit.
+	// For example, '40px' and '40' are both valid values.
+	export let size = '40px';
+	export let padding = '12px';
+	export let color = '';
+	export let bgColor = '';
+	export let border = '1px';
+	export let borderRadius = '2px';
+	export let margin = '20px';
+	// The button will be visible when the user scrolls down
+	// by 'showAt' value.
+	export let showAt = 50;
+
+	// Function for auto appending 'px' unit.
+	function appendPx(value = '') {
```

Comment:
I think this is a bit overkilled hahaha. We won't likely have any use case to allow custom size with custom units and it introduces so many lines of code (At first, I plan to make the component accept no props at all). I think we can keep these props but just accept sizing-related props as a `number` and use it directly with `px` (eg. size 30 px just input `size={30}` without any unit), the same way that the carbon component did. Then we can clean up many unused lines of code.

Reply:
Okie, noted. Will do as you mentioned.